# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/mime/nsMimeTypes.h
# Commit: dc2ad0369b86
# Full Hash: dc2ad0369b8660e3a5e039e03e224122eb46dcfc
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-09 21:19:46
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 9fb8ccb2f305 -r dc2ad0369b86 netwerk/mime/nsMimeTypes.h
--- a/netwerk/mime/nsMimeTypes.h	Tue Apr 09 12:31:55 2024 +0000
+++ b/netwerk/mime/nsMimeTypes.h	Tue Apr 09 12:31:56 2024 +0000
@@ -31,6 +31,7 @@
 #define APPLICATION_GZIP2 "application/gzip"
 #define APPLICATION_GZIP3 "application/x-gunzip"
 #define APPLICATION_BROTLI "application/brotli"
+#define APPLICATION_ZSTD "application/zstd"
 #define APPLICATION_ZIP "application/zip"
 #define APPLICATION_HTTP_INDEX_FORMAT "application/http-index-format"
 #define APPLICATION_ECMASCRIPT "application/ecmascript"
@@ -245,6 +246,7 @@
 #define ENCODING_UUENCODE3 "uuencode"
 #define ENCODING_UUENCODE4 "uue"
 #define ENCODING_YENCODE "x-yencode"
+#define ENCODING_ZSTD "zstd"
 
 /* Some names of parameters that various MIME headers include.
  */